<h1>Table of Contents<span class="tocSkip"></span></h1>
<div class="toc"><ul class="toc-item"><li><span><a href="#Dependencies" data-toc-modified-id="Dependencies-1">Dependencies</a></span></li><li><span><a href="#Functions" data-toc-modified-id="Functions-2">Functions</a></span></li><li><span><a href="#Paths" data-toc-modified-id="Paths-3">Paths</a></span></li><li><span><a href="#Main" data-toc-modified-id="Main-4">Main</a></span></li></ul></div>

# Dependencies

In [1]:
options(scipen = 999)

# Functions


# Paths

In [4]:
datapath <- "/Users/anabbi/OneDrive - UHN/Documents/INDICATE/Data/"
plotpath <- "/Users/anabbi/OneDrive - UHN/Documents/INDICATE/Plots/"

# Main

Create treehouse metadata

In [5]:
clinical <- read.table("~/OneDrive - UHN/Documents/ImmPedCan/treehousegenomics/clinical_TumorCompendium_v11_PolyA_2020-04-09.tsv", sep = "\t",
                      header = TRUE, stringsAsFactors = F)

In [6]:
table(clinical$pedaya)


                                     No             Unknown Yes, age < 30 years 
                425                9426                  82                2814 

In [7]:
pedaya <- clinical[which(clinical$pedaya == "Yes, age < 30 years"),]

In [8]:
pedaya$diseasetype[grepl("lymphoma",pedaya$disease)] <- "Lymphoma"
pedaya$diseasetype[grepl("leukemia",pedaya$disease)] <- "Leukemia"

pedaya$diseasetype[grepl("rhabdoid",pedaya$disease)] <- "Rhabdoid"

#All sarcomas NOS
pedaya$diseasetype[grepl(" sarcoma",pedaya$disease)] <- "SARC"
pedaya$diseasetype[grepl("Ewing",pedaya$disease)] <- "EWS"

pedaya$diseasetype[grepl("neuroblastoma",pedaya$disease)] <- "NBL"
pedaya$diseasetype[grepl("wilms",pedaya$disease)] <- "WILMS"

#pedaya$diseasetype[grepl("sarcoma",pedaya$disease)] <- "SARC" # 5 cases annotated as sarcoma was excluded
pedaya$diseasetype[grepl("osteosarcoma",pedaya$disease)] <- "OS"

#Includes alveolar, embryonal, spindle cell
pedaya$diseasetype[pedaya$disease =="rhabdomyosarcoma"] <- "RMS"
pedaya$diseasetype[grepl("rhabdomyosarcoma",pedaya$disease)] <- "RMS"

In [9]:
pedaya$diseasetype[grepl("adrenocortical carcinoma",pedaya$disease)] <- "Adrenocortical carcinoma"

pedaya$diseasetype[grepl("choroid plexus",pedaya$disease)] <- "Choroid plexus carcinoma"

pedaya$diseasetype[grepl("dysembryoplastic neuroepithelial tumor",pedaya$disease)] <- "Dysembryoplastic neuroepithelial tumor"

pedaya$diseasetype[grepl("ependymoma",pedaya$disease)] <- "Ependymoma"
#Includes ganglioglima and gliomatosis cerebri
pedaya$diseasetype[grepl("glioma",pedaya$disease)] <- "Glioma"
pedaya$diseasetype[grepl("glioblastoma multiforme",pedaya$disease)] <- "Glioblastoma multiforme"

pedaya$diseasetype[grepl("hepatoblastoma",pedaya$disease)] <- "Hepatoblastoma"
#Includes fibrolamellar HCC
pedaya$diseasetype[grepl("hepatocellular carcinoma",pedaya$disease)] <- "Hepatocellular carcinoma"

pedaya$diseasetype[grepl("medulloblastoma",pedaya$disease)] <- "Medulloblastoma"
pedaya$diseasetype[grepl("pheochromocytoma",pedaya$disease)] <- "Pheochromocytoma"

pedaya$diseasetype[grepl("thyroid carcinoma",pedaya$disease)] <- "Thyroid cancer"
pedaya$diseasetype[grepl("thymic carcinoma",pedaya$disease)] <- "Thyroid cancer"

#Includes uveal melanoma
pedaya$diseasetype[grepl("skin cutaneous melanoma",pedaya$disease)] <- "Melanoma"
pedaya$diseasetype[grepl("melanoma",pedaya$disease)] <- "Melanoma"

pedaya$diseasetype[grepl("breast invasive carcinoma",pedaya$disease)] <- "Breast cancer"
pedaya$diseasetype[grepl("cervical & endocervical cancer",pedaya$disease)] <- "Cervical cancer"

pedaya$diseasetype[grepl("testicular germ cell tumor",pedaya$disease)] <- "Testicular germ cell tumor"

pedaya$diseasetype[grepl("supratentorial embryonal tumor NOS",pedaya$disease)] <- "Supratentorial embryonal tumor"

In [10]:
pedaya$diseasetype[is.na(pedaya$diseasetype)] <- "Others"

In [11]:
table(pedaya$disease)


                                                 acinar cell carcinoma 
                                                                     1 
                                                        acute leukemia 
                                                                     4 
                                   acute leukemia of ambiguous lineage 
                                                                     2 
                                          acute lymphoblastic leukemia 
                                                                   548 
                                       acute megakaryoblastic leukemia 
                                                                    93 
                                                acute myeloid leukemia 
                                                                   307 
                                       acute undifferentiated leukemia 
                                                               

In [12]:
pedaya_solid <- pedaya[ pedaya$diseasetype != "Leukemia",]

In [13]:
dim(pedaya_solid)

[1] 1851   10

In [14]:
write.csv(pedaya_solid, file = paste0(datapath, "treehouse_metadata.csv"))

scp the metadata file to samwise ~/immpedcan/wgcna then on immpedcan/4timer2_th/. on samwise, open an R console:

write.csv(tmp_mat_orginal[,1:100], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues100.csv", quote = F)
write.csv(tmp_mat_orginal[,101:200], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues200.csv", quote = F)
write.csv(tmp_mat_orginal[,201:300], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues300.csv", quote = F)
write.csv(tmp_mat_orginal[,301:400], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues400.csv", quote = F)
write.csv(tmp_mat_orginal[,401:500], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues500.csv", quote = F)
write.csv(tmp_mat_orginal[,501:600], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues600.csv", quote = F)
write.csv(tmp_mat_orginal[,601:700], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues700.csv", quote = F)
write.csv(tmp_mat_orginal[,701:800], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues800.csv", quote = F)
write.csv(tmp_mat_orginal[,801:900], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues900.csv", quote = F)
write.csv(tmp_mat_orginal[,901:1000], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues1000.csv", quote = F)
write.csv(tmp_mat_orginal[,1001:1100], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues1100.csv", quote = F)
write.csv(tmp_mat_orginal[,1101:1200], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues1200.csv", quote = F)
write.csv(tmp_mat_orginal[,1201:1300], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues1300.csv", quote = F)
write.csv(tmp_mat_orginal[,1301:1400], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues1400.csv", quote = F)
write.csv(tmp_mat_orginal[,1401:1500], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues1500.csv", quote = F)
write.csv(tmp_mat_orginal[,1501:1600], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues1600.csv", quote = F)
write.csv(tmp_mat_orginal[,1601:1700], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues1700.csv", quote = F)
write.csv(tmp_mat_orginal[,1701:1851], file = "~/immpedcan/4timer2_th/input_timer2_originaltpmvalues1851.csv", quote = F)

